hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -406,5 +405,21 @@
       stage: V2
       tests: [""]
     }
+    {
+      name: target_clock_stretch
+      desc: '''
+            Test clock stretch feature of DUT Target mode.
+            For the write transaction, when acq fifo is full, DUT starts to stretch clock.
+            For the read transaction, when dut receives read command, the tx fifo is empty,
+            DUT starts to stretch clock.
+            Using read / write mixed traffic, trigger stretch condition by slowing down acq / tx
+            fifo process.
```

Comment:
this might not have been evident, but this stretch actually ends up applying to address also. We end up running into a lot of cases where the fifo is so full the address cannot be deposited and has to be stretched as well.

Reply:
Update description line 412 from 'wirte' to 'write and address'